lib/arke/validator.ex

```diff
@@ -173,7 +173,7 @@ defmodule Arke.Validator do
     end
   end
 
-  defp parse_value(%{arke_id: :float} = _, value)
+  defp parse_value(%{arke_id: :float, multiple: false} = _, value)
        when not is_number(value) and not is_nil(value) do
```

Comment:
Maybe only this is needed
```suggestion
       when is_binary(value) do
```